src/routes/visit.routes.js

```diff
@@ -54,33 +55,21 @@ visitRoutes.post('/visit', async (req, res, next) => {
 visitRoutes.put('/visit/:id', async (req, res, next) => {
   try {
     const { id } = req.params
-    const updatedVisit = await visits.update(req.body, { where: { id: id }})
-    res.status(200).send(updatedVisit)
+    const updatedVisit = await visits.update(req.body, { where: { id }})
+    res.status(200).send({updatedVisit})
   } catch (e) {
     console.error(e.message)
   } finally {
     next()
   }
 })
 
-visitRoutes.patch('/visit/:id', async (req, res, next) => {
-  try {
-    const { id } = req.params
-    const updatedVisit = await visits.patch(req.body, { where: { id: id}})
-    res.status(200).send(updatedVisit)
-  } catch(e) {
-    console.error(e.message)
-  } finally {
-    next()
-  }
-})
-
 // TODO: make this a soft delete
 visitRoutes.delete('/visit/:id', async (req, res, next) => {
   try{
     const { id } = req.params
-    const numDeleted = await visits.destroy({ where:{ id: id }})
-    res.status(204).send(numDeleted)
+    const numDeleted = await visits.destroy({ where:{ id }})
+    res.status(200).send({ numDeleted })
```

Comment:
Deletes should return 204 when successful I believe.

Reply:
leaving as 200 to serve a res body